scripts/gen-dumbfs.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+##############################################################################
+##
+##  This file is part of the Chaos Kernel, and is made available under
+##  the terms of the GNU General Public License version 2.
+##
+##  Copyright (C) 2017 - Jakob Kellendonk <jakob.kellendonk@epitech.eu>
+##
+##############################################################################
+
+if (( $# != 2 ))
```

Comment:
I think it's more standard (in bash) to use `if [[ $# -ne 2 ]]`.

Reply:
I think the current version is clearer